project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 // Cinnamon
-//addSbtPlugin("com.lightbend.cinnamon" % "sbt-cinnamon" % "2.12.2")
+//addSbtPlugin("com.lightbend.cinnamon" % "sbt-cinnamon" % "2.12.3")
```

Comment:
Note that this bumps cinnamon to the most recent version at the time this PR is created. It's commented out as cinnamon is enabled 'manually' when commercial modules are used in certain exercises.